### PR TITLE
Updating package descriptions and readmes

### PR DESCRIPTION
**Pull request by karenswry**
Updated all the files and readmes used by external package managers for Themis wrappers (note: this currently excludes Rust as it has its own task).
In some cases, license and readme files were created where they were missing.
JsThemis and Wasm wrappers have now the most breve descriptions in the description files as I had to fit them into a pop-up message since @ilammy finally solved the mystery of where those get displayed in npmjs.com 
Please review and merge.

### CODE REVIEW

File: tools/afl/README.md [hunk 1]
```diff
@@ -136,37 +136,37 @@ Here you can see the following files:
...
+    If the test passes, it exits cleanly.
 
     The exit code does not matter.
-    AFL will react to _abnormal_ behavior of the tool,
+    AFL will react to _abnormal_ behaviour of the tool,
     like crashes and hangups.
-    You can use _abort_() to signal assertion failures,
-    for example.
+    For example, you can use _abort_() to signal
+    assertion failures.
```
[the remark this answers is not in the document]
Comment (karenswry) [reply tools/afl/README.md]: Haha, yep, okay!;)

File: tools/afl/README.md [hunk 2]
```diff
@@ -191,41 +192,41 @@ in order to add a `${new_tool}` to fuzz test suite.
...
-    which can break from malicious or malformed input.
+    that can break from malicious or malformed input.
 
   - Test data should be fairly small (less than 1 KB or so).
 
-    AFL will try minimizing the input size
-    so having big examples does not win you anything.
-    You can use small inputs
-    unless large ones trigger some completely different behavior.
+    AFL will try to minimize the input size
```
[the remark this answers is not in the document]
Comment (karenswry) [reply tools/afl/README.md]: Yeah, you're right. Of course, it is "minimise". Thanks for spotting this!

File: src/wrappers/themis/jsthemis/package.json [hunk 3]
```diff
@@ -1,11 +1,17 @@
 {
   "name": "jsthemis",
   "version": "0.12.0",
-  "description": "Themis is a data security library, providing users with high-quality security services for secure messaging of any kinds and flexible data storage. Themis is aimed at modern developers, with high level OOP wrappers for Ruby, Python, PHP, Java / Android and iOS / OSX. It is designed with ease of use in mind, high security and cross-platform availability.",
+  "description": "Themis is a convenient cryptographic library for data protection.",
```
Comment (vixentael) [inline src/wrappers/themis/jsthemis/package.json]: Do we have any restrictions on description length? 

I thought the goal of updating description in packages was to unify them, and this text is very different from the one we used in `build.gradle` and texts updated by previous PR #587.

Let's make all descriptions consistent

File: src/wrappers/themis/wasm/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "wasm-themis",
   "version": "0.12.1",
-  "description": "Themis data security library",
+  "description": "Themis is a convenient cryptographic library for data protection.",
```
Comment (vixentael) [inline src/wrappers/themis/wasm/package.json]: same as above

File: tools/afl/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (karenswry) [reply tools/afl/README.md]: Changed the spelling.

File: src/wrappers/themis/jsthemis/package.json (hunk 3 above)
Comment (karenswry) [reply src/wrappers/themis/jsthemis/package.json]: Yes, as we've found out with @ilammy, there are
a) noticeable restrictions regarding the overall description size for packages on npmjs.com and the full existing description would not look good and will be cut off abruptly
b) esthetic restrictions regarding the description size - everything that is present in the "description" field in the json file will be crammed into a pop-up next to the package name in the search field on npmjs.com
The solution for this is to have these 2 really breve descriptions for JsThemis and for WasmThemis with the standardised description put into the readme files, featured prominently by npmjs.com.

**Comment by karenswry** [comment]
Implemented the necessary spelling correction and explained the change for JsThemis/WasmThemis descriptions. Can we merge now?

File: src/wrappers/themis/wasm/package.json (hunk 4 above)
Comment (karenswry) [reply src/wrappers/themis/wasm/package.json]: The same explanation as above

File: src/wrappers/themis/jsthemis/package.json (hunk 3 above)
Comment (vixentael) [reply src/wrappers/themis/jsthemis/package.json]: got it, makes sense, thank you!

**Pull request merged**